src/App.jsx

```diff
@@ -1,14 +1,52 @@
 import './App.css';
+import ChatLog from './components/ChatLog';
+import chatData from './data/messages.json';
+import { useState } from 'react';
+
+// Helper function to count total likes
+const calculateTotalLikes = (chatData) => {
+  let total = 0;
+  for (const like of chatData) {
+    total += like.liked;
+  }
+  return total;
+};
 
 const App = () => {
+  const [likeData, setLikeData] = useState(chatData);
```

Comment:
I feel like a more descriptive name for `likeData` would be `chatData` because we have a list of objects that represent messages which means each object is more than just data about whether a message has been liked or not.

Then the setter could also be renamed from `setLikeData` to `setMessageData`.